views/lti/deliverableNavbar.jade

```diff
@@ -26,6 +33,22 @@ script
       navItems.forEach(initializeNavItemEvents)
       navSubTypes.forEach(initializeNavSubTypeEvents)
 
+      const deliverableDropdown = document.querySelector('.deliverable-navbar-header')
```

Comment:
Also, I think it's easier to fix design bugs here if we ever get to have any, than to rewrite stuff and think about refactoring code eventually, so I'd rather stick with specific classes.